server/database/database.js

```diff
@@ -21,13 +21,17 @@ const User = mongoose.model('User', UserSchema);
 
 const MovieSchema = mongoose.Schema({
   title: String,
-
+  img_url: String,
+  hulu: Boolean,
+  disney: Boolean,
+  netflix: Boolean,
```

Comment:
total we will want to have hulu, disney plus, netflix, amazon prime video, hbo max, and Apple TV Plus